README.md

```diff
@@ -0,0 +1,260 @@
+```yaml
+kernel_settings:
+    sysctl:
+        fs.epoll.max_user_watches: 785592
+    operations:
+        sysctl: del
+```
+This will remove whatever `sysctl` settings were set previously, and make
+`fs.epoll.max_user_watches: 785592` the only `sysctl` setting.
+
+The `bootloader` `cmdline` value is a special case.  There may be situations
+where you want to treat it as a list of values, and add or remove specific
+values in the list.  By default, if you specify a value in `kernel_settings`,
+that value will be added to whatever was already there:
+```yaml
+kernel_settings:
+    bootloader:
+        cmdline: "spectre_v2=off nopti quiet splash"
+```
+will result in the values `spectre_v2=off nopti quiet splash` being added to
+the `bootloader` `cmdline` values.  If you want to remove whatever was there,
+and replace it with your specified values, use `operations` with `del` like
+this:
+```yaml
+kernel_settings:
+    bootloader:
+        cmdline: "spectre_v2=off nopti quiet splash"
+    operations:
+        bootloader:
+        - {"del": "cmdline"}
+```
+then the values `spectre_v2=off nopti quiet splash` will *replace* the
+existing `cmdline` values, if any.  If you want to remove specific values, use
+`del` and specify the values you want to remove:
+```yaml
+kernel_settings:
+    ...
+    operations:
+        bootloader:
+        - {"del": "cmdline", "value": "spectre_v2=off nopti"}
+```
+if the previous value was `spectre_v2=off nopti quiet splash`, the items
+`spectre_v2=off` and `nopti` will be removed from the list, and the final
+value will be `quiet splash`.
+Also note that you can have conflicts.  For example, if the previous value was
+`spectre_v2=off nopti quiet splash` and you tried to add the value
+`spectre_v2=on`, you would get a `Conflict` warning because you specified the
+key `spectre_v2` with a different, conflicting value.  If you want to replace
+`spectre_v2=off` with `spectre_v2=on`, use an `operations` with a `del` to
+remove the values you want to get rid of, then in your `kernel_settings`
+`bootloader` `cmdline` specify the values that you want to keep:
+```yaml
+kernel_settings:
+    bootloader:
+        cmdline: "spectre_v2=on nopti quiet splash"
+    ...
+    operations:
+        bootloader:
+        - {"del": "cmdline", "value": "spectre_v2=off"}
+```
+
+## Dependencies
+
+None
+
+## Example Playbook
+
+Including an example of how to use your role (for instance, with variables
+passed in as parameters) is always nice for users too:
+
+```yaml
+- hosts: all
+  vars:
+    template_foo: "foo foo!"
+    template_bar: "progress bar"
+
+  roles:
+    - linux-system-roles.template
+```
+
+## License
+
+Some parts related to `tuned` are `GPLv2+`.  These are noted in the headers
+of the files.  Everything else is `MIT`, except where noted.  See the file
+`LICENSE` for more information.
+
+## Author Information
+
+Rich Megginson (richm on github, rmeggins at my company)
+
+## Testing
```

Comment:
While it is certainly useful to have testing docs, I don't think they should be part of the user docs (README.md) as the audience is different. @t-woerner any opinion?

Reply:
moved to README-testing.md